### PR TITLE
Feat: Make pod start time available als logging link template variable in kubeflow plugin

### PR DESCRIPTION
**Pull request by fg91**
# TL;DR

I'd like to add the filter `timestamp >= podStartTime` to our stackdriver log link template. This is important because currently our users have to manually extend the time range (default 1h) each time they visit the stackdriver log page. (Unless the job started and finished within the past hour.)

To this aim, I recently made the pod start and finish time available as logging link template variables in RFC3339 time format which is required by google cloud logging (https://github.com/flyteorg/flyteplugins/pull/360).

Currently, the kubeflow plugin doesn't pass the pod start time to the log plugin though.

In this PR I extend the kubeflow plugin to do so.

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [x] Any pending items have an associated Issue

## Complete description
_NA_

## Tracking Issue
_NA_

## Follow-up issue
_NA_

### CODE REVIEW

File: go/tasks/plugins/k8s/kfoperators/common/common_operator.go [hunk 1]
```diff
@@ -104,8 +105,11 @@ func GetMPIPhaseInfo(currentCondition commonOp.JobCondition, occurredAt time.Tim
 }
 
 // GetLogs will return the logs for kubeflow job
-func GetLogs(taskType string, name string, namespace string, hasMaster bool,
+func GetLogs(taskType string, objectMeta meta_v1.ObjectMeta, hasMaster bool,
```
Comment (fg91) [inline go/tasks/plugins/k8s/kfoperators/common/common_operator.go:108]: By passing in `objectMeta` as opposed to individually passing in the name, namespace and in the future also the creation timestamp, we can retrieve the creation timestamp only a single time here instead of doing so in `pytorch.go`, `tensorflow.go`, as well as `mpi.go`.

**Review by hamersaw — APPROVED** [review]
